pkgs/development/compilers/dotnet/default.nix

```diff
@@ -69,18 +85,16 @@ makeScopeWithSplicing' {
 
       dotnet_8 = recurseIntoAttrs (callPackage ./8 { });
       dotnet_9 = recurseIntoAttrs (callPackage ./9 { });
+
+      sdk_8_0 = dotnet_8.sdk;
+      sdk_8_0_1xx = dotnet_8.sdk;
+      runtime_8_0 = dotnet_8.runtime;
+      aspnetcore_8_0 = dotnet_8.aspnetcore;
```

Comment:
Doesn't the binary release of the .NET 8 SDK have a more recent band than the source built one?
If that's the case, I believe `sdk_8_0` should point to the most recent release which is the binary one.